keps/sig-node/3063-dynamic-resource-allocation/README.md

```diff
@@ -1162,6 +1163,18 @@ Once all of those steps are complete, kubelet will notice that the claims are
 ready and run the pod. Until then it will keep checking periodically, just as
 it does for other reasons that prevent a pod from running.
 
+### Handling non graceful node shutdowns
```

Comment:
```suggestion
### Handling non-graceful node shutdowns
```

Reply:
'non graceful' is used in the KEP, that's why I decided to use it here and in the e2e test code.

Reply:
The KEP issues uses "non-graceful", as does the KEP README in one place - looks like the original authors weren't sure about the right spelling.

"non-graceful" feels more right to me, but I'm not a native speaker and English is creative... Let's leave it in this PR as you have it now ("non graceful").